src/sidebar/components/Thread.js

```diff
@@ -205,7 +205,15 @@ function Thread({ thread, threadsService }) {
             data-testid="thread-children"
           >
             {visibleChildren.map(child => (
-              <li key={child.id} className="mt-2">
+              <li
+                key={child.id}
+                className={classnames(
+                  'mt-2',
+                  // Ensure correct rendering of replies with RTL content.
+                  // See https://github.com/hypothesis/client/issues/4705.
+                  '[text-align:start]'
```

Comment:
Tailwind doesn't have a built-in utility for this ([related issue](https://github.com/tailwindlabs/tailwindcss/issues/949)), and I don't know if we'll need it anywhere else yet, so I used https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties to define custom properties.

Reply:
Another option would be to add `style` to this element, but I think what you've done here is fine. The lack of RTL Tailwind classes has already come up once before, darnit!